src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/winrt/DispatcherTaskExtensions.cs

```diff
@@ -0,0 +1,54 @@
+﻿//----------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//------------------------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using Windows.UI.Core;
+
+namespace Microsoft.IdentityModel.Clients.ActiveDirectory.Internal.Platform
+{
+    internal static class DispatcherTaskExtensions
+    {
+        public static async Task<T> RunTaskAsync<T>(this CoreDispatcher dispatcher,
+            Func<Task<T>> func, CoreDispatcherPriority priority = CoreDispatcherPriority.High)
```

Comment:
>CoreDispatcherPriority.High [](start = 66, length = 27)

does this have to be High?

Reply:
no, it should be normal, good catch, will change

Reply:
great. approving. assuming it will be addressed before commit.

---
In reply to: [167149844](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/982#discussion_r167149844) [](ancestors = 167149844)